pkg/api/v1beta3/dynakube/dynakube_types.go

```diff
@@ -198,6 +202,8 @@ type TemplatesSpec struct {
 	// Low-level configuration options for the LogModule feature.
 	// +kubebuilder:validation:Optional
 	LogModule LogModuleTemplateSpec `json:"logModule,omitempty"`
+	// +kubebuilder:validation:Optional
+	KspmNodeConfigurationCollector KspmNodeConfigurationCollectorSpec `json:"kspmNodeConfigurationCollector,omitempty"`
```

Comment:
nitpick: this name is bit wrong (IMO)

According to the epic:
> Deploy the K8s Node Configuration Collector Daemon Set with the Dynatrace Operator to enable Kubernetes Security Posture Management (KSPM) use cases.

So KSPM is the feature/usecase  -> so the `KspmSpec` makes sense, as it's the feature you turn off/on

But the **component** we deploy is the `K8s Node Configuration Collector` -> so in the `templates` section, we should name it after the **component** the user will configure, something like `K8sNodeConfigurationCollector`

wdyt?

cc @chrismuellner

Reply:
Examples:
- We don't call the `OpenTelemetryCollector` the `ExtensionsOpenTelemetryCollector`, even though it only affects the `extensions` feature
- `LogModule` is fine, because we deploy the **component** called `LogModule`
- Same for  `ExtensionExecutionController`, as that is the name of the **component**

Reply:
I agree, but if we want this changes we need to discuss again with Product so we are all on the same page. These are customer facing changes you are proposing after all

Reply:
outcome: leave it as is for now, will discuss it further with stakeholders, and rename if necessary